From 02f2154913abf0581b0b0bd1f5d192346e48a650 Mon Sep 17 00:00:00 2001
From: Adam Seaton <aseaton@google.com>
Date: Fri, 12 Oct 2018 13:42:41 -0700
Subject: [PATCH 05/10] [DO NOT MERGE]Update Security String to 2018-12-05 for
 December release bug: 117667960 (cherry picked from commit
 1e6d1d603badde694db0e1ce748088c431a328ae)

Change-Id: I8434e940ea45b3d133670b67e16e8cb2224f2323
---
 core/version_defaults.mk | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/core/version_defaults.mk b/core/version_defaults.mk
index e8edf6a09..f837260d6 100644
--- a/core/version_defaults.mk
+++ b/core/version_defaults.mk
@@ -231,7 +231,7 @@ ifndef PLATFORM_SECURITY_PATCH
     #  It must be of the form "YYYY-MM-DD" on production devices.
     #  It must match one of the Android Security Patch Level strings of the Public Security Bulletins.
     #  If there is no $PLATFORM_SECURITY_PATCH set, keep it empty.
-      PLATFORM_SECURITY_PATCH := 2018-12-01
+      PLATFORM_SECURITY_PATCH := 2018-12-05
 endif
 
 ifndef PLATFORM_SECURITY_PATCH_TIMESTAMP
-- 
2.17.1

